dialogue-serde/src/main/java/com/palantir/conjure/java/dialogue/serde/ExceptionDeserializingErrorDecoder.java

```diff
@@ -188,6 +172,46 @@ private RuntimeException decodeInternal(Response response) {
         }
     }
 
+    private RuntimeException jsonExceptionFromBody(byte[] body, int code) throws IOException {
```

Comment:
nit:
```suggestion
    private RemoteException jsonExceptionFromBody(byte[] body, int code) throws IOException {
```

Just in case we mess with this implementation later, would be good to have the type-checker warn us if we try to throw anything other than a `RemoteException`.